docs/application/native/guides/app-management/app-controls.md

```diff
@@ -335,16 +341,15 @@ Regardless of the launch request type, when the application launcher framework h
 
-    It takes 4 parameters and uses them to initialize the application. The `argc` and `argv` parameters contain the values from the application framework, and you must never change their values. The third parameter is a state transition handler that is responsible for managing the state transitions the application goes through while it is running. The fourth parameter is application data to be passed to each state handler.
+    It takes four parameters and uses them to initialize the application. The `argc` and `argv` parameters contain the values from the application framework, and these values must remain unchanged. The third parameter is a state transition handler that is responsible for managing the state transitions the application goes through, while it is running. The fourth parameter is application data that is passed to each state handler.
 
-3. When the `ui_app_main()` is first invoked, the application moves from the ready state to the created state, and must initialize itself. During this transition, the application framework calls the application's `app_create_cb()` state transition callback just before the application enters the main loop. Within the registered callback, you must initialize the application resources and create the main window.
+3. When the `ui_app_main()` is invoked, the application moves from the ready state to the created state, and must initialize itself. During this transition, the application framework calls the application's `app_create_cb()` state transition callback just before the application enters the main loop. Within the registered callback, you must initialize the application resources and create the main window.
 
     If the `app_create_cb()` callback function returns `false`, the application moves to the terminated state. If it returns `true`, the application enters the main loop.
 
+4. Just after the application enters the main loop, the application framework calls the 'app_control_cb()' callback.
 
-4. Just after the application enters the main loop, the application framework calls the application's `app_control_cb()` callback.
-
-    The callback is passed to the `app_control` handle containing the reason why the application was launched. For example, the application can be launched to open a file to handle the request that has been sent by other application. The application is always responsible for checking the content of the `app_control` handle and responding appropriately. The content of the `app_control` handle can be empty, if the application is launched by the user from the launcher.
+    For example, the application can be launched to open a file to handle the request that is sent by other application. The application is always responsible for checking the content of the app_control handle and respond appropriately. The content of the app_control handle can be empty, if the application is launched by the user from the launcher.
```

Comment:
@mijin85cho I didn't request to delete the "The callback is passed to the app_control handle containing the reason why the application was launched." line. You may check my comment for the same below, as it has mentioned the total rephrase for the paragraph and "The callback is passed to..." is present in the rephrase:

"Line 349, 350, 351:
Kind rephrase the lines as
"Just after the application enters the main loop, the application framework calls the `app_control_cb()` callback.
The callback is passed to the `app_control handle` containing the reason why the application was launched.

For example, the application can be launched to open a file to handle the request that is sent by other application. The application is always responsible for checking the content of the app_control handle and respond appropriately. The content of the `app_control` handle can be empty, if the application is launched by the user from the launcher.""

I changed 'responding' to 'respond' because 'responding' sounds like a continuous form, means where action is happening right now, whereas 'respond' sounds like a simple form.

Reply:
@rishu-samsung About "responding": My point was the grammar of the sentence.
When it is changed into 'respond', if we check the skeleton of the sentence, it becomes
 "A is responsible for checking and respond".
Is this usage correct grammatically?